astropy/units/quantity_helper/scipy_special.py

```diff
@@ -71,7 +71,6 @@ def get_scipy_special_helpers():
 
     SCIPY_HELPERS = {}
     for name in dimensionless_to_dimensionless_sps_ufuncs:
-        # In SCIPY_LT_1_5, erfinv and erfcinv are not ufuncs.
         ufunc = getattr(sps, name, None)
         if isinstance(ufunc, np.ufunc):
```

Comment:
@mhvk, do you know if we still need the `isinstance(ufunc, np.ufunc)` check? And related to that, is the following still needed? https://github.com/astropy/astropy/blob/e65f9756448f65fe0ac2aa5d4ee6f4399ab56adb/astropy/units/tests/test_quantity_ufuncs.py#L1527-L1530

Reply:
No, we don't -- thanks for catching that. More generally, I really should not have introduced those kinds of checks... Much better just to use version numbers as those can be found much more easily.

Reply:
I pushed another commit with what I think you wanted. Please re-review. Thanks!